### PR TITLE
Bulk delete refactor

### PR DESCRIPTION
**Pull request by nopjmp**
Also add an editorconfig and globally handle unhandledRejection messages to get rid of a deprecation warnings and give a better stacktrace.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -16,5 +16,8 @@ client.on('message', (message) => handleMessage(message, config, client.commands
 client.on('ready', () => client.user.setActivity(config.activity.name, { type: config.activity.type }));
 
 client.on('error', (error) => console.log(error));
+process.on('unhandledRejection', function (reason, p) {
```
Comment (flutterflies) [inline src/index.js]: Is there a reason to not use an arrow function here?

File: src/utils.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+exports.deleteMessages = function (channel, deleteAmount) {
```
Comment (flutterflies) [inline src/utils.js]: An arrow function again here?

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = false
```
Comment (flutterflies) [inline .editorconfig]: I think this is something we'd want set to true?

Comment (nopjmp) [reply .editorconfig]: This was just a default, but I'll set it to true.

File: src/utils.js (hunk 2 above)
Comment (nopjmp) [reply src/utils.js]: Arrow function only changes the "this" context, but I'll change it to an arrow function for readability.

File: src/index.js (hunk 1 above)
Comment (nopjmp) [reply src/index.js]: either way works, I'll switch it to an arrow function.

**Pull request merged**